landing-pages/site/assets/scss/_ol-ul.scss

```diff
@@ -0,0 +1,80 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+@import "typography";
+@import "colors";
+
+ol, ul {
```

Comment:
Oh. The UL and LI element is also used in many other places. We should use this selector carefully.
<img width="762" alt="Screenshot 2019-10-10 at 21 10 18" src="https://user-images.githubusercontent.com/12058428/66598907-626c3a00-eba2-11e9-9eed-d7d3b37bd842.png">

Reply:
Oops, alright, I'll use class instead of tag

Reply:
done